### ISSUE TITLE
When cloning a recipe, the new recipe should be disabled

### ISSUE THREAD

**Issue by benoit74** [openzim/zimfarm#898]
When we clone a recipe, the clone keep the original recipe status in terms of "activation", i.e. if the recipe was enabled, the clone is enabled.

This shouldn't be the case because the clone needs by design to be updated before being requested, so it should be disabled automatically so that the scheduler might not pick it for scheduling.

**Comment by kelson42** [comment]
Pretty sure 5is was working that way in the past

**Comment by rgaudin** [comment]
It has always worked like this. Definitely a regression. Could be pg-migration related but I believe I've cloned properly after that

**Comment by benoit74** [comment]
2 senior on this topic vs 1 junior, I can only admit you are right 🤣 
Let's add a `prio 1` tag then.

**Comment by Mightymanh** [comment]
Hello may I get assigned to this issue? I have set up development environment and attempt to clone a recipe from an existing recipe using admin priviledge. Is "enable status" referred to the status in the box drawn in the image below?

<img width="860" alt="Image" src="https://github.com/user-attachments/assets/0e07506f-3cf0-4f97-82ee-a442ad0333a9" />

**Comment by benoit74** [comment]
Yes it is. No matter the original recipe status (enabled or not), the clone must be disabled after the initial cloning operation (done with the Clone button at the top).

**Comment by Mightymanh** [comment]
@benoit74 I created a pull request. You can review it.

**Comment by benoit74** [comment]
Please do not ping me in issue but rather in PR when it's ready, otherwise I get two notifications, one for the issue and one for the PR while it is the same matter.
